src/main/java/Duke/ListStorage.java

```diff
@@ -0,0 +1,89 @@
+package Duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListStorage {
+    /** Single List to store Tasks */
+    public List<Task> myTasks;
+    String SPACING = "    ";
```

Comment:
I think this constant can be declared as `static final`, considering its use case.